swaybar/ipc.c

```diff
@@ -13,9 +13,10 @@
 void ipc_send_workspace_command(struct swaybar *bar, const char *ws) {
 	const char *fmt = "workspace \"%s\"";
 	uint32_t size = snprintf(NULL, 0, fmt, ws);
-	char command[size];
+	char *command = malloc(sizeof(char) * size);
```

Comment:
`snprintf` returns the number of chars printed, excluding the terminating NULL byte. This bug was there before, but we need to add `+ 1` here.